pkg/controller/backup_sync_controller.go

```diff
@@ -179,26 +185,25 @@ func (c *backupSyncController) run() {
 
 		for backupName := range backupStoreBackups {
 			log = log.WithField("backup", backupName)
-			log.Debug("Checking backup store backup to see if it needs to be synced into the cluster")
+			log.Debug("Checking this backup to see if it needs to be synced into the cluster")
 
 			// use the controller's namespace when getting the backup because that's where we
 			// are syncing backups to, regardless of the namespace of the cloud backup.
 			backup, err := c.backupClient.Backups(c.namespace).Get(backupName, metav1.GetOptions{})
 			if err == nil {
 				log.Debug("Backup already exists in cluster")
 
-				if backup.Spec.StorageLocation != "" {
-					continue
-				}
-
 				// pre-v0.10 backups won't initially have a .spec.storageLocation so fill it in
-				log.Debug("Patching backup's .spec.storageLocation because it's missing")
-				if err := patchStorageLocation(backup, c.backupClient.Backups(c.namespace), location.Name); err != nil {
-					log.WithError(err).Error("Error patching backup's .spec.storageLocation")
+				if backup.Spec.StorageLocation == "" {
```

Comment:
Should we drop this code? 1.0.0 wasn't supposed to support pre-v0.10 backups.

Reply:
Yeah I think that's fine, we probably just missed it bc there was no // TODO(1.0) on it